messaging/firebase-messaging-sw.js

```diff
@@ -5,7 +5,7 @@ importScripts('/__/firebase/4.8.1/firebase-app.js');
 importScripts('/__/firebase/4.8.1/firebase-messaging.js');
 importScripts('/__/firebase/init.js');
 
-const messaging = firebase.messaging();
```

Comment:
The switch here is debatable because FWIW any browser that supports service workers are also gonna support es6 syntax, but I guess it's fine for consistency.

Reply:
Good point.  Maybe we revisit the ES6 decision but as it stands it's easier to just be consistent.